change/@ni-nimble-components-8fdb7efd-10a8-4e79-9dd7-079527db6644.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Can be patch since table is still incubating
```suggestion
  "type": "patch",
```

Reply:
Man, I keep doing this.